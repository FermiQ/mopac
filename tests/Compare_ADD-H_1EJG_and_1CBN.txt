#
# Script for use with the HTML file "Compare_ADD-H_1EJG_and_1CBN.html"
#
