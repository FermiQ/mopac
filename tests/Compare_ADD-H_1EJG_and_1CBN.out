 *******************************************************************************
 ** Site#:    0         For non-commercial use only           Version 21.055M **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.055M, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web-site: HTTP://OpenMOPAC.net.          **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Feb 26 19:06:23 2021  *
 *  COMPARE    - Compare two geometries
 *  GEO_DAT    - DATA SET GEOMETRY IS IN FILE "PDB1EJG.arc"
 *  GEO_REF    - REFERENCE GEOMETRY IS IN FILE "PDB1CBN.arc"
 *               (NO BIAS TOWARDS THE REFERENCE GEOMETRY WILL BE APPLIED)
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *  OUTPUT     - MINIMIZE OUTPUT
 *******************************************************************************
geo_dat="PDB1EJG.arc" geo_ref="PDB1CBN.arc" compare output
 Two PDB structures for Crambin (Atoms in PRO/SER 22, LEU/ILE 25, and others deleted)



                            Differences in atoms sets

          Atoms in GEO_DAT only                   Atoms in GEO_REF only

   1   ATOM    305  N   PRO A  22              ATOM    305  N   SER A  22
   2   ATOM    306  CA  PRO A  22              ATOM    306  CA  SER A  22
   3   ATOM    307  C   PRO A  22              ATOM    307  C   SER A  22
   4   ATOM    308  O   PRO A  22              ATOM    308  O   SER A  22
   5   ATOM    309  CB  PRO A  22              ATOM    309  CB  SER A  22
   6   ATOM    310  CG  PRO A  22              ATOM    310  OG  SER A  22
   7   ATOM    311  CD  PRO A  22              ATOM    311  H   SER A  22
   8   ATOM    312  HA  PRO A  22              ATOM    312  HA  SER A  22
   9   ATOM    313 1HB  PRO A  22              ATOM    313 1HB  SER A  22
  10   ATOM    314 2HB  PRO A  22              ATOM    314 2HB  SER A  22
  11   ATOM    315 1HG  PRO A  22              ATOM    315  HG  SER A  22
  12   ATOM    316 2HG  PRO A  22              ATOM    342  N   ILE A  25
  13   ATOM    317 1HD  PRO A  22              ATOM    343  CA  ILE A  25
  14   ATOM    318 2HD  PRO A  22              ATOM    344  C   ILE A  25
  15   ATOM    345  N   LEU A  25              ATOM    345  O   ILE A  25
  16   ATOM    346  CA  LEU A  25              ATOM    346  CB  ILE A  25
  17   ATOM    347  C   LEU A  25              ATOM    347  CG1 ILE A  25
  18   ATOM    348  O   LEU A  25              ATOM    348  CG2 ILE A  25
  19   ATOM    349  CB  LEU A  25              ATOM    349  CD1 ILE A  25
  20   ATOM    350  CG  LEU A  25              ATOM    350  H   ILE A  25
  21   ATOM    351  CD1 LEU A  25              ATOM    351  HA  ILE A  25
  22   ATOM    352  CD2 LEU A  25              ATOM    352  HB  ILE A  25
  23   ATOM    353  H   LEU A  25              ATOM    353 1HG1 ILE A  25
  24   ATOM    354  HA  LEU A  25              ATOM    354 2HG1 ILE A  25
  25   ATOM    355 1HB  LEU A  25              ATOM    355 1HG2 ILE A  25
  26   ATOM    356 2HB  LEU A  25              ATOM    356 2HG2 ILE A  25
  27   ATOM    357  HG  LEU A  25              ATOM    357 3HG2 ILE A  25
  28   ATOM    358 1HD1 LEU A  25              ATOM    358 1HD1 ILE A  25
  29   ATOM    359 2HD1 LEU A  25              ATOM    359 2HD1 ILE A  25
  30   ATOM    360 3HD1 LEU A  25              ATOM    360 3HD1 ILE A  25
  31   ATOM    361 1HD2 LEU A  25              HETATM  641  C1  EOH A  66
  32   ATOM    362 2HD2 LEU A  25              HETATM  642  O   EOH A  66
  33   ATOM    363 3HD2 LEU A  25              HETATM  643  C2  EOH A  66
  34                                           HETATM  644  H   EOH A  66
  35                                           HETATM  645 1H1  EOH A  66
  36                                           HETATM  646 2H1  EOH A  66
  37                                           HETATM  647 1H2  EOH A  66
  38                                           HETATM  648 2H2  EOH A  66
  39                                           HETATM  649 3H2  EOH A  66

     If "LET" is used, then some or all of these differences will be ignored

                      Number of atoms in GEO_DAT:  642
                      Number of atoms in GEO_REF:  648
          Number of atoms common to both systems:  609


                        After docking

                     Atoms that move a lot

                     Atom Label             GEO_REF Coordinates       Movement    Integral

            (ATOM     92 2HD1 ILE A   7)  10.757  -0.683  15.934        3.59         3.59
            (ATOM    607 1HD2 ASN A  46)  11.403   3.861  16.203        2.53         6.13
            (ATOM     87 2HG1 ILE A   7)  11.195   0.949  13.414        2.50         8.63
            (ATOM    608 2HD2 ASN A  46)  12.578   5.003  16.395        2.36        10.99
            (ATOM    600  ND2 ASN A  46)  11.916   4.601  15.746        2.30        13.28
            (ATOM    609  HE2 GLU A  23)  13.484  11.051   0.529        2.20        15.48
            (ATOM    599  OD1 ASN A  46)  13.443   3.218  14.924        2.18        17.66
            (ATOM     93 3HD1 ILE A   7)  11.247  -1.323  14.347        1.70        19.37
            (ATOM     82  CD1 ILE A   7)  11.293  -0.460  15.011        1.67        21.03
            (ATOM     91 1HD1 ILE A   7)  12.334  -0.233  15.241        1.28        22.31
            (ATOM    515 1HG2 THR A  39)  24.001  13.400  13.356        1.01        23.33
            (ATOM    561  HD2 ASP A  43)  12.483  15.777  18.818        1.01        24.34
            (ATOM    556  OD2 ASP A  43)  12.244  14.902  19.239        0.93        25.27
            (ATOM    517 3HG2 THR A  39)  23.483  11.849  12.654        0.92        26.19
            (ATOM    516 2HG2 THR A  39)  22.288  12.921  13.423        0.82        27.01
            (ATOM    510  CG2 THR A  39)  23.199  12.887  12.825        0.77        27.78
            (ATOM    560 2HB  ASP A  43)  11.704  12.468  19.392        0.69        28.48
            (ATOM    554  CG  ASP A  43)  13.083  14.013  18.900        0.66        29.13
            (ATOM     90 3HG2 ILE A   7)   9.069  -1.372  13.432        0.63        29.76
            (ATOM    559 1HB  ASP A  43)  13.091  12.607  20.499        0.63        30.39
            (ATOM    555  OD1 ASP A  43)  14.052  14.180  18.165        0.62        31.01
            (ATOM     80  CG1 ILE A   7)  10.659   0.726  14.337        0.59        31.60
            (ATOM    509  OG1 THR A  39)  24.152  13.569  10.774        0.58        32.18
            (ATOM     86 1HG1 ILE A   7)  10.705   1.589  15.001        0.53        32.70
            (ATOM    166 1HD2 ASN A  12)   0.422   4.102   9.755        0.49        33.19
            (ATOM    102  HA  VAL A   8)   5.334   1.282  11.688        0.49        33.68
            (ATOM    553  CB  ASP A  43)  12.776  12.653  19.456        0.47        34.15
            (ATOM    167 2HD2 ASN A  12)   1.788   3.528  10.480        0.43        34.58
            (ATOM    161  ND2 ASN A  12)   1.432   4.110   9.735        0.40        34.99
            (ATOM    513  HB  THR A  39)  22.749  14.628  11.666        0.39        35.38

     Residues that move a lot and their (Movement in Angstroms per atom)

    Ile   7(0.80)    Asn  46(0.63)    Asp  43(0.43)    Thr  39(0.41)    Glu  23(0.25)
    Asn  12(0.22)    Gly  37(0.17)    Val   8(0.16)    Pro  36(0.15)    Ala  38(0.12)

   Difference between GEO_DAT and GEO_REF: 77.360 = total,  0.1270 = Average,  0.3333 = RMS, in Angstroms

                             Differences between bond-lengths for the two geometries

           Diff.                 Atom 1                       Atom 2                  Bond length      Bond length
                                                                                      in GEO_DAT      in GEO_REF
   1       0.133     C(ATOM    553  CB  ASP A  43)    C(ATOM    554  CG  ASP A  43)      1.634            1.501
   2       0.091     C(ATOM    365  CA  TYR A  29)    C(ATOM    368  CB  TYR A  29)      1.492            1.584
   3       0.089     O(ATOM    599  OD1 ASN A  46)    C(ATOM    598  CG  ASN A  46)      1.219            1.309
   4       0.082     O(ATOM    509  OG1 THR A  39)    C(ATOM    508  CB  THR A  39)      1.478            1.395
   5       0.082     C(ATOM     95  CA  VAL A   8)    C(ATOM     98  CB  VAL A   8)      1.530            1.613
   6       0.082     N(ATOM    600  ND2 ASN A  46)    C(ATOM    598  CG  ASN A  46)      1.327            1.245
   7       0.082     C(ATOM    439  CB  ILE A  34)    C(ATOM    441  CG2 ILE A  34)      1.607            1.525
   8       0.081     O(ATOM    313  OE2 GLU A  23)    C(ATOM    311  CD  GLU A  23)      1.216            1.297
   9       0.069     C(ATOM     80  CG1 ILE A   7)    C(ATOM     82  CD1 ILE A   7)      1.573            1.504
  10       0.064     C(ATOM    505  CA  THR A  39)    C(ATOM    508  CB  THR A  39)      1.541            1.605
  11       0.057     O(ATOM     21  OG1 THR A   2)    C(ATOM     20  CB  THR A   2)      1.452            1.396
  12       0.049     N(ATOM     40  N   CYS A   4)    C(ATOM     32  C   CYS A   3)      1.341            1.292
  13       0.048     C(ATOM    477  CB  PRO A  36)    C(ATOM    478  CG  PRO A  36)      1.541            1.492
  14       0.044     C(ATOM     51  CA  PRO A   5)    C(ATOM     52  C   PRO A   5)      1.524            1.480
  15       0.043     O(ATOM    556  OD2 ASP A  43)    C(ATOM    554  CG  ASP A  43)      1.312            1.269
  16       0.040     C(ATOM    550  CA  ASP A  43)    C(ATOM    551  C   ASP A  43)      1.540            1.580
  17       0.039     C(ATOM    407  CA  CYS A  32)    C(ATOM    408  C   CYS A  32)      1.547            1.508
  18       0.039     N(ATOM    364  N   TYR A  29)    C(ATOM    365  CA  TYR A  29)      1.466            1.427
  19       0.039     N(ATOM    549  N   ASP A  43)    C(ATOM    544  C   GLY A  42)      1.335            1.374
  20       0.038     O(ATOM     78  O   ILE A   7)    C(ATOM     77  C   ILE A   7)      1.255            1.217
  21       0.037     N(ATOM    350  N   THR A  28)    C(ATOM    351  CA  THR A  28)      1.485            1.447
  22       0.037     C(ATOM    455  CA  ILE A  35)    C(ATOM    458  CB  ILE A  35)      1.539            1.576
  23       0.036     O(ATOM    294  O   THR A  21)    C(ATOM    293  C   THR A  21)      1.232            1.269
  24       0.036     C(ATOM    550  CA  ASP A  43)    C(ATOM    553  CB  ASP A  43)      1.539            1.503
  25       0.036     N(ATOM     30  N   CYS A   3)    C(ATOM     31  CA  CYS A   3)      1.442            1.478

    GEO_DAT:  'PDB1EJG.arc'
    GEO_REF:  'PDB1CBN.arc'


                    Analysis of Non-Covalent Interactions

          Total non-covalent energy of GEO_DAT system:   -527.29 Kcal/mol
          Total non-covalent energy of GEO_REF system:   -527.40 Kcal/mol
                                           Difference:      0.11 Kcal/mol

          Number of hydrogen bonds common to both systems:   54


    The following files will be written:
              'Compare_ADD-H_1EJG_and_1CBN.html'
From GEO_REF: 'Compare_ADD-H_1EJG_and_1CBN_PDB1CBN.pdb'
From GEO_DAT: 'Compare_ADD-H_1EJG_and_1CBN_PDB1EJG.pdb'

          GEO_REF with 0SCF: Job complete

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * GEO_REF with 0SCF: Job complete                                            *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.01 SECONDS

 == MOPAC DONE ==
